components/dashboard/sections/HostDashboardHostedCollectives.tsx

```diff
@@ -33,6 +35,7 @@ const schema = z.object({
   searchTerm: searchFilter.schema,
   orderBy: orderByFilter.schema,
   hostFeesStructure: z.nativeEnum(HostFeeStructure).optional(),
+  type: z.nativeEnum(HostedCollectiveTypes).optional(),
```

Comment:
Since the GraphQL argument `accountType` is a list, we could allow the user to pick many types in the filter also, by using the `isMulti` helper from `lib/filters/schemas`:

```suggestion
  type: isMulti(z.nativeEnum(HostedCollectiveTypes)).optional(),
```